src/openrct2-win/openrct2-win.cpp

```diff
@@ -48,33 +48,29 @@ int wmain(int argc, wchar_t** argvW, [[maybe_unused]] wchar_t* envp)
 static char** GetCommandLineArgs(int argc, wchar_t** argvW)
 {
     // Allocate UTF-8 strings
-    auto argv = (char**)malloc(argc * sizeof(char*));
-    if (argv != nullptr)
+    auto argv = new char*[argc];
+    //char** argv = (char**)malloc(argc * sizeof(char*));
```

Comment:
Please remove this comment. (Git allows us to see or roll back changes if needed.)